Api/Models/Request/AuthenticationRequestModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Diagnosea.Submarine.Api.Models.Request
+{
+    public class AuthenticationRequestModel
+    {
+        [Required]
+        public string PrivateSigningKey { get; set; }
+        [Required]
+        public string Audience { get; set; }
+        [Required] 
+        public string Issuer { get; set; }
```

Comment:
Guess this'll come from the sender URL -- I think this can be retrieved from headers but not 100%.